src/register/RegistrationPage.jsx

```diff
@@ -115,7 +115,7 @@ class RegistrationPage extends React.Component {
     if (this.tpaHint) {
       payload.tpa_hint = this.tpaHint;
     }
-    payload.is_registered = this.isRegistered;
+    payload.is_register_page = this.isRegistered;
```

Comment:
Do we need this `this.isRegistered` anywhere else? If not can we remove this and simply assign the `True` here?

Reply:
Updated.